cmake/FindGMP.cmake

```diff
@@ -115,6 +115,14 @@ if(NOT GMP_FOUND_SYSTEM)
     endif()
   endif()
 
+  # On Windows, we need to have a shell interpreter to call 'configure'
+  if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
+      find_program (SHELL "sh" REQUIRED)
+      message(STATUS "Found shell interpreter: ${SHELL}")
+  else()
+      set(SHELL "")
+  endif()
+
```

Comment:
We could move this check into `cmake/deps-helper.cmake` and also use it in the ANTLR file. There are potentially more scripts that will require `SHELL`:

- `cmake/FindCoCoA.cmake`
- `cmake/FindCLN.cmake`
- `cmake/FindANTLR3.cmake`

Reply:
Done.